benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/SignatureBase.java

```diff
@@ -38,7 +40,9 @@ protected void setup(
     signer.update(message);
     signature = signer.sign();
     verifier.update(message);
-    assert verifier.verify(signature);
```

Comment:
note to other reviewers -- we found that this `assert` not only wasn't enforced in the benchmark build, but it also swallowed up checked exceptions from this call! so, we shouldn't use the `assert` keyword in the  benchmark.